docs/dev/Build.md

```diff
@@ -555,9 +555,9 @@ sudo chmod 0600 /etc/siodb/instances/siodb/config
 sudo chown siodb:siodb /etc/siodb/instances/siodb/config
 sudo -u siodb dd if=/dev/random of=/etc/siodb/instances/siodb/master_key bs=16 count=1
 sudo chmod 0600 /etc/siodb/instances/siodb/master_key
-sudo cp config/sample_keys/rsa /etc/siodb/instances/siodb/encryption.master_key
-sudo chmod 0600 /etc/siodb/instances/siodb/encryption.master_key
-sudo chown siodb:siodb /etc/siodb/instances/siodb/encryption.master_key
+sudo cp config/sample_keys/rsa.pub /etc/siodb/instances/siodb/initial_access_key
```

Comment:
destination file name must be `master_key`, not `initial_access_key`
fix all occurrences

Reply:
@cxxman `master_key` is for the encryption `initial_access_key` is for the initial public key added to `root` user. If I rename `initial_access_key` to `master_key`, this suite of command won't work. Isn't it?

Reply:
yes, really